alpine/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM alpine:latest
+MAINTAINER Peter Martini <PeterCMartini@GMail.com>
+
+RUN mkdir -p /usr/src/perl
+COPY *.patch /usr/src/perl/
+WORKDIR /usr/src/perl
+
+ENV PERL_CPANM_OPT --no-wget
+
+RUN set -x \
+    && NPROC=$(getconf _NPROCESSORS_ONLN) \
+    && apk add --no-cache --virtual .build-deps curl procps tar build-base make \
+    && curl -SL https://cpan.metacpan.org/authors/id/S/SH/SHAY/perl-5.22.1.tar.bz2 -o perl-5.22.1.tar.bz2 \
+    && echo '29f9b320b0299577a3e1d02e9e8ef8f26f160332 *perl-5.22.1.tar.bz2' | sha1sum -c - \
+    && tar --strip-components=1 -xjf perl-5.22.1.tar.bz2 -C /usr/src/perl \
+    && rm perl-5.22.1.tar.bz2 \
+    && cat *.patch | patch -p1 \
+    && ./Configure -Duse64bitall -Duseshrplib  -des \
```

Comment:
This could benefit from the same multi-arch tweaks I added to #43 (just add `dpkg-dev` and `dpkg` to the `.build-deps` installation above and you can even use the same exact code), although note that https://github.com/gliderlabs/docker-alpine/issues/304 is the blocker for adding proper multiarch in the official images for Alpine-based variants.

Reply:
(This comment is definitely still relevant :eyes:)

Reply:
Yep, I was checking if we could do this with just `getconf` and friends, but this is easier.